src/screens/Customers/EditCustomer.tsx

```diff
@@ -15,28 +25,106 @@ const styles = (theme: Theme) =>
     content: {
       color: theme.palette.text.primary,
     },
+    formControl: {
+      width: '100%',
+    },
   });
 
 interface EditCustomerProps extends RouteComponentProps {
-  classes: { header: string; content: string };
+  classes: { header: string; content: string; formControl: string; };
   location: any;
+  currentCustomer: CustomerRecord;
+  currentSite: SiteRecord;
+  user: User;
 }
 
 function EditCustomer(props: EditCustomerProps) {
-  const { classes } = props;
+  const { classes, currentCustomer, currentSite, user } = props;
+  const history = useHistory();
+
+  const [selectedTariffPlanId, setSelectedTariffPlanId] = useState(currentCustomer.tariffPlanId);
+  const [customerName, setCustomerName] = useState(currentCustomer.name);
+  const [reason, setReason] = useState("");
+  const [hasMeter, setHasMeter] = useState(currentCustomer.hasmeter);
+  const [customerInactive, setCustomerInactive] = useState(currentCustomer.isactive);
+
+  const handleSelectTariffPlan = (event: React.ChangeEvent<{ value: unknown }>) => {
+    setSelectedTariffPlanId(event.target.value as string);
+  }
+
+  // TODO: Add form input validation and error messaging
+  const handleSubmit = (event: React.MouseEvent) => {
+    // Prevent page refresh on submit
+    event.preventDefault();
+
+    // Add other info necessary to create the Airtable record
+    let customerUpdate: CustomerUpdateRecord = {
+      id: '',
+      dateUpdated: formatUTCDateStringToLocal((new Date()).toString()),
+      customerIds: [currentCustomer.id],
+      explanation: reason,
+      userId: user.id,
+    };
+
+    // Make a deep copy of an empty customer record
+    const customer: CustomerRecord = JSON.parse(JSON.stringify(currentCustomer));
+    customer.name = customerName;
+    customer.hasmeter = hasMeter;
+    customer.isactive = !customerInactive;
+    customer.tariffPlanId = selectedTariffPlanId;
+
+    if (customer.customerUpdates === undefined) {
+      customer.customerUpdates = [customerUpdate];
+    } else {
+      customer.customerUpdates.push(customerUpdate);
+    }
+
+    // TODO: add error handling
+    editCustomerInRedux(customer);
```

Comment:
Actually, this `editCustomerInRedux` call might even be better placed inside of `editCustomer`. Then, we can abstract away the `editCustomer` function as BOTH a frontend change and a backend change and ensure they're consistent since there should never be a case where I update the frontend and not the backend and vice verase. 

This applies to `addCustomerToRedux` as well. Thoughts on this? @wangannie